Add GitHub Sponsors to Donate Page, Minor Added 2023 to copyright in footer (#165)
pulsar-edit/pulsar-edit.github.io: docs/donate.md

  <img src="https://opencollective.com/webpack/donate/button@2x.png?color=blue" width=300 />
</a>

> Additionally if you'd rather contribute to the running costs of the project via
> [GitHub Sponsors](https://github.com/sponsors/pulsar-edit) you can visit our GitHub profile to safely donate.

Reviewer: ```suggestion
If you'd rather contribute via [GitHub Sponsors](https://github.com/sponsors/pulsar-edit)
you can do this on our GitHub profile.
```

Just being picky as always, just thinking to avoid repetition a little to make it read a little easier.

Author: Thanks a ton, love the recommendation
